webapp/app/assessment/fra/sections/forestPolicy/sectionSpec.js

```diff
@@ -12,6 +12,7 @@ const variableMappings = {
 
 const tableSpec = SectionSpec.newTableSpec({
   [SectionSpec.KEYS_TABLE.name]: section.tables.forestPolicy,
+  [SectionSpec.KEYS_TABLE.columnsExport]: ['national', 'subnational'],
```

Comment:
do you handle the internationalization of the labels later?

Reply:
yes, i will have mapping